pkg/services/dashboardversion/dashverimpl/dashver.go

```diff
@@ -38,33 +38,34 @@ type Service struct {
 	cfg       *setting.Cfg
 	store     store
 	dashSvc   dashboards.DashboardService
-	k8sclient client.K8sHandler
+	k8sclient dashboardclient.K8sHandlerWithFallback
 	features  featuremgmt.FeatureToggles
 	log       log.Logger
 }
 
-func ProvideService(cfg *setting.Cfg, db db.DB, dashboardService dashboards.DashboardService, dashboardStore dashboards.Store, features featuremgmt.FeatureToggles,
-	restConfigProvider apiserver.RestConfigProvider, userService user.Service, unified resource.ResourceClient, dual dualwrite.Service, sorter sort.Service) dashver.Service {
+func ProvideService(
```

Comment:
Probably something for a followup PR: It seems that a lot of stuff in this function input was needed to initialise the `k8sclient` - since we don't need them anymore, should we remove them?

Reply:
Done!